<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpCameraDialog</class>
 <widget class="QDialog" name="IpCameraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_IpCamera_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>132</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frameIpCameraSetting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_IpCamera_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>87</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditUrl5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>267</y>
     <width>308</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>url5</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_IpCamera_2">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>42</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditUrl2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>132</y>
     <width>308</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>url2</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName3">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>177</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name3</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditUrl3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>177</y>
     <width>308</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>url3</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditUrl6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>312</y>
     <width>308</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>url6</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_IpCamera_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>42</y>
     <width>308</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>URL地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName4">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>222</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name4</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName1">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>87</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name1</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName5">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>267</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name5</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName2">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>132</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name2</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_IpCamera_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>177</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditUrl4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>222</y>
     <width>308</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>url4</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>网络摄像机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_IpCamera_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>222</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_IpCamera_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>312</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditUrl1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>87</y>
     <width>308</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>url1</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName6">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>312</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name6</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_IpCamera_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>267</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frameIpCameraSetting</zorder>
  <zorder>label_IpCamera_4</zorder>
  <zorder>label_IpCamera_1</zorder>
  <zorder>lineEditUrl5</zorder>
  <zorder>label_IpCamera_2</zorder>
  <zorder>lineEditUrl2</zorder>
  <zorder>lineEditName3</zorder>
  <zorder>lineEditUrl3</zorder>
  <zorder>lineEditUrl6</zorder>
  <zorder>label_IpCamera_3</zorder>
  <zorder>lineEditName4</zorder>
  <zorder>lineEditName1</zorder>
  <zorder>lineEditName5</zorder>
  <zorder>lineEditName2</zorder>
  <zorder>label_IpCamera_5</zorder>
  <zorder>lineEditUrl4</zorder>
  <zorder>label_title</zorder>
  <zorder>label_IpCamera_6</zorder>
  <zorder>label_IpCamera_8</zorder>
  <zorder>lineEditUrl1</zorder>
  <zorder>lineEditName6</zorder>
  <zorder>label_IpCamera_7</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
